api/hub/package.json

```diff
@@ -64,6 +64,7 @@
     "node-abort-controller": "3.0.0",
     "object-hash": "2.2.0",
     "prom-client": "13.2.0",
+    "ts-node": "^10.2.1",
```

Comment:
no need for this dependency

Reply:
This is needed by the worker loader file (the js one).